pinning-services/index.js

```diff
@@ -1,6 +1,8 @@
 const cron = require('cron');
 const fetch = require('node-fetch');
 const fs = require('fs');
+const httpPublisherIpv4Address = '198.50.215.6';
+const httpPublisherIpv6Address = '2607:5300:203:4014:5074:f2ff:feb1:a87f';
```

Comment:
These need to come in from main config, add to https://github.com/hyphacoop/api.distributed.press/blob/main/backend/config.template.json

Reply:
Added to config and now reads from that file